status.go

```diff
@@ -142,6 +153,23 @@ func (self *StatusMoniter) Add(ID, containerName string) {
 	self.Apps[ID] = app
 	Metrics.Add(ID, app)
 	Lenz.Attacher.Attach(ID, app)
+	reportContainerCure(ID)
+}
+
+func isInWatchingSet(cid string) bool {
+	conn, err := common.Rds.Acquire()
+	if err != nil || conn == nil {
+		logs.Assert(err, "Get redis conn")
+	}
+	defer common.Rds.Release(conn)
+
+	containersKey := fmt.Sprintf("eru:agent:%s:containers", config.HostName)
+	rep, err := gore.NewCommand("SISMEMBER", containersKey, cid).Run(conn)
+	if err != nil {
+		logs.Assert(err, "Get targets")
+	}
+	repInt, _ := rep.Int()
+	return repInt == 1
```

Comment:
[this](https://github.com/keimoon/gore/pull/3) is why I don't use `repBool, _ := rep.Bool()` ...